src/components/ec-letter-icon/ec-letter-icon.vue

```diff
@@ -26,6 +27,7 @@ import { type LetterIconProps, LetterIconSize } from './types';
 const props = withDefaults(defineProps<LetterIconProps>(), {
   size: LetterIconSize.SMALL,
   isClickable: false,
+  applyHoverStyles: false,
```

Comment:
nit picking here, but since is effectively a boolean should be `is` or `has`

So something like `isHoverable` or `hasHoverEnabled` or similar

Reply:
Sure, let's go with `isParentHovered` since the purpose of the prop is to trigger the hover styles when the parent is hovered